src/test/components/__snapshots__/TimelineMarkers.test.js.snap

```diff
@@ -83,15 +83,102 @@ Array [
     1,
   ],
   Array [
```

Comment:
looks like the previous snapshot was wrong here, because the thread wasn't consistent :) see below